assets/quick-add-bulk.js

```diff
@@ -33,6 +33,10 @@ class QuickAddBulk extends HTMLElement {
     }
   }
 
+  setInput() {
+    this.input = this.querySelector('quantity-input input');
```

Comment:
Can we do 
```
getInput() {
    return this.querySelector('quantity-input input');
}
```

and then do `this.getInput().addEvent.....` or wherever else it is used.

Why do we want to store it in a variable?

Reply:
Thanks @utkarshsaxena93 I need a bit time to learn this approach and figure out how it works but looks interesting even for my development. I'll take a look :)